Singular.js

```diff
@@ -8,24 +8,29 @@ const SDK_VERSION = version;
 
 export class Singular {
 
-    static _singularLinkHandlerEmitter = new NativeEventEmitter(SingularBridge);
-
+    static _singularNativeEmitter = new NativeEventEmitter(SingularBridge);
+    
     static init(singularConfig) {
-        if (!singularConfig.singularLinkHandler) {
-            SingularBridge.init(singularConfig.apikey, singularConfig.secret, singularConfig.customUserId, singularConfig.sessionTimeout);
-        } else {
-            this._singularLinkHandler = singularConfig.singularLinkHandler;
+        this._singularLinkHandler = singularConfig.singularLinkHandler;
+        this._conversionValueHandler = singularConfig.conversionValueHandler;
+
+        SingularBridge.init(JSON.stringify(singularConfig))
 
-            this._singularLinkHandlerEmitter.addListener(
+        this._singularNativeEmitter.addListener(
                 'SingularLinkHandler',
                 singularLinkParams => {
                     if (this._singularLinkHandler) {
                         this._singularLinkHandler(singularLinkParams);
                     }
                 });
 
-            SingularBridge.initWithSingularLink(singularConfig.apikey, singularConfig.secret, singularConfig.customUserId, singularConfig.sessionTimeout);
-        }
+        this._singularNativeEmitter.addListener(
+                'ConversionValueHandler',
+                conversionValue => {
+                    if (this._conversionValueHandler) {
+                        this._conversionValueHandler(conversionValueHandler);
```

Comment:
I think this should be conversionValue instead conversionValueHandler

Reply:
true

Reply:
please change it my love